### PR TITLE
Missing german translation for logout confirmation #11961

### LINKED ISSUE #11961: Missing german translation for logout confirmation
### Describe the bug

Logout page is missing german translations

### Version

18.0.0

### Expected behavior

Logout confirmation page is translated in german

### Actual behavior

Logout confirmation is displayed in english, even if german local is selected

### How to Reproduce?

Add German locale to realm supported locales

Login with a client and logout with the /logout URL, but without an idTokenHint, the logout confirmation is displayed in english

### Anything else?

PR is coming

### PR DESCRIPTION
**Pull request by PhilippStein**
Fixes #11961

### CODE REVIEW

File: themes/src/main/resources-community/theme/base/login/messages/messages_de.properties
Comment (DGuhr) [inline themes/src/main/resources-community/theme/base/login/messages/messages_de.properties:383]: Could you please change this to: "Zugriff verweigert"? instead of "verboten"? More common.

**Review by DGuhr — CHANGES_REQUESTED** [review]
Apart from "Zugriff verboten" LGTM, thanks :) could you please change that to the more common "Zugriff verweigert"? Danke ;)

File: themes/src/main/resources-community/theme/base/login/messages/messages_de.properties
Comment (PhilippStein) [reply themes/src/main/resources-community/theme/base/login/messages/messages_de.properties]: Hups, makes sense, changed

Comment (DGuhr) [reply themes/src/main/resources-community/theme/base/login/messages/messages_de.properties]: Thanks. 👍 @abstractj could you please trigger the CI run for this one, then? thanks.

**Review by DGuhr — APPROVED** [review]

**Comment by abstractj** [comment]
@DGuhr @PhilippStein Tausend dank! Workflow approved and once it finishes we should be able to merge.

**Pull request merged**
